src/components/WithdrawPage.tsx

```diff
@@ -179,18 +178,19 @@ const WithdrawPage = (props: Props): ReactElement => {
           </div>
           <AdvancedOptions />
 
-          <Button
-            variant="contained"
-            sx={{ width: "240px" }}
-            disabled={
-              noShare ||
-              !!formStateData.error ||
-              formStateData.lpTokenAmountToSpend.isZero()
-            }
-            onClick={onSubmit}
-          >
-            {t("withdraw")}
-          </Button>
+          <Box display="flex" justifyContent="center" width="100%" mt={5}>
```

Comment:
using `width="100%"` is dangerous as it will exceed the width of the containing element. https://i.stack.imgur.com/MTNBw.png

can you leverage FlexBox here instead to achieve the same result? It will probably involve changing flex settings in the parent elements and having some other containers to make sure it all cascades together correctly: https://mui.com/system/flexbox/ 

https://css-tricks.com/snippets/css/a-guide-to-flexbox/